packages/core/src/hooks/useSupportConfig.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useApi, configApiRef } from '@backstage/core-api';
+
+export function useSupportConfig() {
+  const configApi = useApi(configApiRef);
+
+  return {
+    title: configApi.getOptionalString('app.support.title') ?? '#backstage',
```

Comment:
is your use of `#backstage` here to attempt to be a bit backward compatible?  i would have thought this might have been straight English a la just "Backstage", as in the title of the support URL. (which on line 26 is not a link to slack)

Reply:
This no longer does this here, but I added a `Discord Chatroom` item in the `app-config.yaml`. It's currently set to `#backstage` for the link title. I'm looking for better ideas on the name.